Description: Redirect mksnapshot log to stdout
 armel builds typically fail at mksnapshot, for which it is useful to be able to get the actual log.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2011-10-25
--- a/tools/gyp/v8.gyp
+++ b/tools/gyp/v8.gyp
@@ -166,7 +166,7 @@
           'variables': {
             'mksnapshot_flags': [
               '--log-snapshot-positions',
-              '--logfile', '<(INTERMEDIATE_DIR)/snapshot.log',
+              '--logfile', '-',
             ],
             'conditions': [
               ['v8_random_seed!=0', {
@@ -267,7 +267,7 @@
           'variables': {
             'mksnapshot_flags': [
               '--log-snapshot-positions',
-              '--logfile', '<(INTERMEDIATE_DIR)/snapshot.log',
+              '--logfile', '-',
             ],
             'conditions': [
               ['v8_random_seed!=0', {
